tests/operators/test_presto_to_mysql.py

```diff
@@ -0,0 +1,37 @@
+import unittest
```

Comment:
The file is missing Apache license

```
Could not find Apache license headers in the following files:
 !????? /app/tests/operators/test_presto_to_mysql.py
```

Reply:
@kaxil oh you are right. Thank you. Will fix this tomorrow.